core/src/main/scripts/importer/validateData.py

```diff
@@ -3632,6 +4075,9 @@ def main_validate(args):
     relaxed_mode = False
     if hasattr(args, 'relaxed_clinical_definitions') and args.relaxed_clinical_definitions:
         relaxed_mode = True
+    strict_mode = False
+    if hasattr(args, 'strict_mutation_checks') and args.strict_mutation_checks:
+        strict_mode = True
```

Comment:
I'm more of a fan of using `default=False` in the parser.add_argument, and then using `strict_mode = args.strict_mutation_checks`